egs2/RESPIN/asr1/conf/tuning/train_conformer_ctc.yaml

```diff
@@ -0,0 +1,61 @@
+batch_type: numel
```

Comment:
It is good to have various configs, but for start, we would recommend to be concise (e.g., 2 or 3 configs would be better).

Reply:
Let's keep the one that we really use.

Reply:
sure, right now its all default libri files, I'll change it